traffic-policy/actions/forward-internal/behavior.mdx

```diff
@@ -0,0 +1,23 @@
+## Behavior
+
+When the Forward Internal action executes, it will look up the specified endpoint and relay any incoming traffic. The endpoint being forwarded to will behave as if the traffic was sent directly to it and execute all of the actions in its associated Traffic Policy. The endpoint being forwarded to must exist in the same ngrok account and be the same protocol as the forwarding endpoint (e.g. an HTTP public endpoint can only forward to an HTTP private endpoint). You also may not forward traffic to a second private endpoint or back to the original endpoint in a loop.
+
+If the forwarding is successful, the response from the upstream for the private endpoint will be sent back to the client making the original request. No further actions in the Traffic Policy will be executed and no traffic will be sent to the upstream for the public endpoint.
```

Comment:
do we run the `on_http_response` phase on the public forwarding endpoint still?

Reply:
Oh good point, yes. I'll add that in.